### ISSUE TITLE
Issue for the first running

### ISSUE THREAD

**Issue by thangvuongbk** [expectocode/telegram-export#63]
Hello all,

this is my first running this application. and firstly, thanks for great work to community,
Unfortunately, I am currently getting the below error
---------------------
```
2018-06-28 01:08:02,066 - telethon.network.mtprotosender - WARNING - Attempt 1 at new auth_key failed: 
2018-06-28 01:08:02,066 - telethon.network.mtprotosender - WARNING - Attempt 1 at new auth_key failed: 
2018-06-28 01:08:02,278 - telethon.network.mtprotosender - WARNING - Attempt 2 at new auth_key failed: 
2018-06-28 01:08:02,278 - telethon.network.mtprotosender - WARNING - Attempt 2 at new auth_key failed: 
2018-06-28 01:08:02,491 - telethon.network.mtprotosender - WARNING - Attempt 3 at new auth_key failed: 
2018-06-28 01:08:02,491 - telethon.network.mtprotosender - WARNING - Attempt 3 at new auth_key failed: 
2018-06-28 01:08:02,701 - telethon.network.mtprotosender - WARNING - Attempt 4 at new auth_key failed: 
2018-06-28 01:08:02,701 - telethon.network.mtprotosender - WARNING - Attempt 4 at new auth_key failed: 
2018-06-28 01:08:02,911 - telethon.network.mtprotosender - WARNING - Attempt 5 at new auth_key failed: 
2018-06-28 01:08:02,911 - telethon.network.mtprotosender - WARNING - Attempt 5 at new auth_key failed: 
Traceback (most recent call last):
  File "D:\03_Trading\04_Python\telegram-export\telegram_export\__main__.py", line 307, in <module>
    ret = loop.run_until_complete(main(loop)) or 0
  File "C:\Python36\lib\asyncio\base_events.py", line 468, in run_until_complete
    return future.result()
  File "D:\03_Trading\04_Python\telegram-export\telegram_export\__main__.py", line 280, in main
    ).start(config['TelegramAPI']['PhoneNumber']))
  File "C:\Python36\lib\site-packages\telethon\client\auth.py", line 116, in _start
    await self.connect()
  File "C:\Python36\lib\site-packages\telethon\client\telegrambaseclient.py", line 276, in connect
    self.session.server_address, self.session.port)
  File "C:\Python36\lib\site-packages\telethon\network\mtprotosender.py", line 125, in connect
    await self._connect()
  File "C:\Python36\lib\site-packages\telethon\network\mtprotosender.py", line 272, in _connect
    raise e
ConnectionError: auth_key generation failed 5 times
```
---------------------
 any one let me know the solution for this?
thank you very much in advance.

**Comment by Lonami** [comment]
Please format the issue correctly.

**Comment by Lonami** [comment]
It was a bug for some people which should be fixed in `telethon` v1.0.2, so please upgrade and try again.

**Comment by Lonami** [comment]
It should no longer be an issue if you're using the latest version. If it is you should provide `DEBUG` logs.
